### ISSUE TITLE
#189 broke lodash with mixins

### ISSUE THREAD

**Issue by andreineculau** [wix/eslint-plugin-lodash#191]
I'm afraid #189 has broken cases of lodash usage with mixins

After 9288ad494397e0f6d7df6ea212b5022737599dae I can get errors like

```
TypeError: Cannot read property 'prop' of undefined
    at Object.methodSupportsShorthand (/Users/andrei/.../node_modules/eslint-plugin-lodash/lib/util/methodDataUtil.js:74:113)
    at always (/Users/andrei/.../node_modules/eslint-plugin-lodash/lib/util/lodashUtil.js:128:32)
    at /Users/andrei/.../node_modules/eslint-plugin-lodash/lib/util/lodashUtil.js:91:13
```

I have inferred the problem is with the mixins, because the function that fails will actually get called with `mapValuesDeep`, which is actually a mixin from lodash-firecloud https://github.com/tobiipro/lodash-firecloud/blob/master/src/map-values-deep.js

(you can install lodash-firecloud via `npm i git://github.com/tobiipro/lodash-firecloud#v0.1.5`)

**Comment by andreineculau** [comment]
i should note that the eslint-plugin-lodash configuration is here https://github.com/tobiipro/eslint-config-firecloud/blob/master/configs/lodash.js

**Comment by ganimomer** [comment]
@andreineculau Thanks for reporting!
Fixed in `v5.0.1`.

**Comment by andreineculau** [comment]
~~insert swear word~~ thanks! that was fast!
